lang_python.adoc

```diff
@@ -13,98 +13,138 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
 `-t python` option:
 
 [source,shell]
-kaitai-struct-compiler -t python my_spec.ksy
+kaitai-struct-compiler -t python --python-package . my_spec.ksy
 
 This normally generates `my_spec.py` parser module, which is compatible
-with both Python 2 and 3.
+with both Python 2 and 3. It is strongly recommended to use python 3
+because python 2 is EOL since 2020-01-01.
+
+`--python-package` optional CLI arg allows you to select a parent package
+name used by one generated file to import symbols from another one.
+
+If you want to embed KSC-generated files into an own package, it is
+recommended to set the arg to `.`, which means "current package",
+rather than to the real  name of the subpackage, because setting it
+to a real name will cause breakage when you (or anyone else) want
+to reorganize or rename your package. It will cause KSC generate
+relative imports instead of absolute ones for the specs imported into
+your main spec:
+
+----
+from .your_imported_spec import YourImportedSpec
+----
+
+(note the dot we have passed as an arg) instead of
+
+----
+from your_imported_spec import YourImportedSpec
+----
+.
+
+**Without setting this arg at all you will likely get an `ImportError`,
+if any of the specs imports another spec.** You can instead set this
+arg to the absolute path of the submodule, but it opens a can of worms.
+
+It is also recommended to put all the KSC-generated code into an
+own subpackage, in example `kaitai`, so, if your build process
+regenerates the files on every build, you could just .gitignore the dir.
 
-TODO: document `--python-package`
 
 [[add-runtime]]
 === Adding runtime library
 
 Code generated by Kaitai Struct compiler will require additional runtime
-library (https://pypi.org/project/kaitaistruct/[kaitaistruct at PyPi],
+library (https://pypi.org/project/kaitaistruct/[image:https://img.shields.io/pypi/v/kaitaistruct.svg[kaitaistruct at PyPi]],
 https://github.com/kaitai-io/kaitai_struct_python_runtime[sources at
 GitHub]).
 
 Runtime library API version must match compiler version that you're using
-(i.e. if you use ksc v0.8, then you must use compatible v0.8 runtime).
+(i.e. if you use ksc v0.9, then you must use compatible v0.9 runtime).
+
+If you are using snapshot KSC, you should also use snapshot runtime.
 
 [[add-runtime-release]]
 ==== Using release version
 
-With stable version of Kaitai Struct compiler, one can use use following
+With stable version of Kaitai Struct compiler, one can use the following
 methods:
 
 * On Debian / Ubuntu, one can install it from apt repositories:
 
-** For Python 2.x:
-+
+** For Python 3.x:
+
```

Comment:
You should add here the list continuation sign `+` here (and on a few other places), because right now the shell code block and the reference links below break the continuity of the list item "For Python 3.x:". Therefore you can see the sudden decrease of indentation where the `sudo apt-get` code block starts, when it should be horizontally aligned with the list item bullet instead. 

All paragraphs that belong to a particular list item are separated with the plus sign `+` instead of the blank line in Asciidoc, see [Asciidoctor quick reference](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#lists), the last example _Complex content in outline lists_. 

```suggestion
+
```

Reply:
Done, thanks.